lua/feline/init.lua

```diff
@@ -8,6 +8,19 @@ local M = {}
 
 M.reset_highlights = gen.reset_highlights
 
+function M.update_all_windows()
+    for _, winid in ipairs(api.nvim_list_wins()) do
+        vim.wo[winid].statusline = api.nvim_win_call(winid,
+            function()
+                return require('feline').statusline(winid)
```

Comment:
No need to pass winid

Reply:
Actually, it might be better to add winid as a parameter to statusline and use that to compare to `g.statusline_winid`